model/Core/Vocabularies/RelationshipType.md

```diff
@@ -67,9 +67,9 @@ between a Package and a File, between two Packages, or between one SPDXDocument
 - patch: Every `to` Element is a patch for the `from` Element
 - prerequisite: Every `to` Element is a prerequisite of the `from` Element
 - providedDependency: Every `to` Element is a dependency not included in the distributed artifact but is assumed to be provided the `from` Element
-- publishedBy: (Security) Designates the agent that made a vulnerability record available for public use or reference
-- reportedBy: (Security) Designates the agent that first reported a vulnerability to the project, vendor, or tracking database for formal identification 
-- republishedBy: (Security) Designates the agent that tracked, aggregated, and/or enriched vulnerability details to improve context (i.e. NVD)
+- publishedBy: (Security) Designates a `from` Vulnerability element was made available for public use or reference by each `to` Agent
```

Comment:
Probably should just refer to "Vulnerability". rather than "Vulnerability element"?